src/bun.js/bindings/exports.zig

```diff
@@ -1839,6 +1832,26 @@ pub const ZigConsoleClient = struct {
                     this.formatter.writeIndent(Writer, this.writer) catch {};
                 }
 
+                fn getObjectName(this: *@This(), globalThis: *JSC.JSGlobalObject, value: JSValue) ZigString {
+                    _ = this;
+                    var name_str = ZigString.init("");
+                    value.getNameProperty(globalThis, &name_str);
+                    if (name_str.len > 0 and !strings.eqlComptime(name_str.slice(), "Object")) {
+                        return name_str;
+                    }
+
+                    value.getPrototype(globalThis).getNameProperty(globalThis, &name_str);
+                    if (name_str.len > 0 and !strings.eqlComptime(name_str.slice(), "Object")) {
+                        return name_str;
+                    }
+
+                    value.getClassName(globalThis, &name_str);
+                    if (name_str.len > 0 and !strings.eqlComptime(name_str.slice(), "Object")) {
+                        return name_str;
+                    }
+                    return ZigString.init("");
```

Comment:
Not sure if this is the best way. I have a check `if (name_str.len > 0) {` that I would like to fail if we reach this case.

Reply:
use ZigString.static here

Reply:
actually these functions are nearly identical. init is fine here.